### DISCUSSION TITLE
Add an option to restrict elytra looting from end ships/end cities.

### DISCUSSION THREAD

**Discussion by ghost** [PurpurMC/Purpur#962] [Feature Requests]
On a lot of survival servers, elytras get looted by players that want to make a fortune out of selling them, adding an option to restrict these players from looting X amount of elytras in X time, would make it easier for other players to retrieve their own elytra!

**Comment by YouHaveTrouble** [comment]
due to how elytras are spawned in item frames, that might be quite hard to do.
Even if possible, this would probably be way easier to do as a plugin.

**Comment by ApliNi** [comment]
paper has a feature that automatically replenishes chests (and elytra, also prevents one person from taking many times.
However, it requires that the container has not been destroyed. If someone digs the treasure chest, it cannot be replenished.

**Reply by YouHaveTrouble** [reply]
> paper has a feature that automatically replenishes chests (and elytra, also prevents one person from taking many times. However, it requires that the container has not been destroyed. If someone digs the treasure chest, it cannot be replenished.

Chests have something that item frames do not - loot tables. Paper replenishes chests based on the existance of one, that's why it stops when you destroy the chest.
